kcbq-connector/test/docker/connect/connect-docker.sh

```diff
@@ -20,5 +20,5 @@ connect-standalone \
     /etc/kafka-connect-bigquery/standalone.properties \
     /etc/kafka-connect-bigquery/connector.properties &
 
-sleep 60
+sleep 180
```

Comment:
nit: suggest leave a comment to help understand

Reply:
Added comments accordingly~